components/lvgl/lv_conf.h

```diff
@@ -833,6 +833,11 @@ typedef void * lv_font_user_data_t;
 #  define LV_SPRINTF_INCLUDE <stdio.h>
 #  define lv_snprintf     snprintf
 #  define lv_vsnprintf    vsnprintf
+#else   /*!LV_SPRINTF_CUSTOM*/
```

Comment:
What do you think about doing something like this:
```c
#if defined CONFIG_LVGL_SPRINTF_DISABLE_FLOAT
#define LV_SPRINTF_DISABLE_FLOAT 0
#else
#define LV_SPRINTF_DISABLE_FLOAT 1
#endif
```

Reply:
Yes, that's better. But we need to swap 0 and 1, it should read

```
#else   /*!LV_SPRINTF_CUSTOM*/
#  if defined CONFIG_LVGL_SPRINTF_DISABLE_FLOAT
#    define LV_SPRINTF_DISABLE_FLOAT 1
#  else
#    define LV_SPRINTF_DISABLE_FLOAT 0
#  endif
```

You want me to change the PR? (I'm not familiar with this suggestion function yet...)